ProjectApplication/project_core/serializers.py

```diff
@@ -37,19 +39,33 @@ class Meta:
         fields = ('latitude', 'longitude', )
 
 
+class FilterMediumSerializer(serializers.ListSerializer):
+    def to_representation(self, data):
+        data = data.filter(~Q(license_id=None))
+        return super(FilterMediumSerializer, self).to_representation(data)
```

Comment:
Feel free to use `super().to_representation(data)` - less typing / easier refactoring :-)